packages/cli/src/index.js

```diff
@@ -97,6 +106,78 @@ config({
   multiline: true,
 })
 
+const TELEMETRY_ENABLED =
+  telemetry !== 'false' && !process.env.REDWOOD_DISABLE_TELEMETRY
```

Comment:
The env var is more important here / takes precedence right? The logic doesn't really read that way. I know it's a bit tough since the env var can be undefined, but maybe this should just be a couples lines instead of one?

Reply:
I hadn't really considered precedence here. I guess if you are so specific that you pass it in as an argument to the command line you're being pretty strict so maybe that does take precedence?

Either way this logical and checks for both cases. If we switched the order then we'd short circuit on an `undefined`, as you highlighted, and would not get the behaviour we desired. Happy to rework it to be less concise in an effort to be more readable.

Reply:
Yeah we have to consider it; if they both specify different things, one has to win. I thought env vars were considered more specific than flags but let's find a precedent

Reply:
Ah yeah you're right, don't have a definitive resource, but it seems like flags win out, and the way our code already works is to give precedence to flags, like with RWJS_CWD here: https://github.com/redwoodjs/redwood/blob/a001cff7ad45acca8a5d940890462cdd5b7e70aa/packages/cli/src/index.js#L59-L60